cli/internal/ffi/proto/messages.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.12
+// 	protoc        v3.21.7
```

Comment:
I don't think we're supposed to commit this. 
cc @arlyon

Reply:
My bad, this sneaked it. It keeps popping up again, I think this diff might be missing on main?